calico/openstack/t_etcd.py

```diff
@@ -17,10 +17,25 @@
 
 # Etcd-based transport for the Calico/OpenStack Plugin.
 
+# Standard Python library imports.
```

Comment:
This comment is weird, probably because it's really close to saying "Python standard library imports", which many of these aren't.